test/evergreen.yml

```diff
@@ -2574,6 +2596,29 @@ buildvariants:
     - name: static-wt-build-test
     - name: format-failure-configs-test
 
+- name: ubuntu1804-asan
+  display_name: "! Ubuntu 18.04 ASAN"
+  run_on:
+  - ubuntu1804-wt-build
```

Comment:
Not sure if the ASAN tasks need to be run on `ubuntu1804-wt-build` distro, which is only used by those compiler tasks as this builder has various versions of GCC/Clang preinstalled. If not, please use the normal `ubuntu1804-test` distro instead.